vtex-apps/shelf-components
# Add view and product click events

#### What is the purpose of this pull request?


#### What problem is this solving?


#### How should this be manually tested?

- [Workspace](https://rec--storecomponents.myvtex.com/apparel---accessories/)
- go to the bottom of the page
- in the `window.pixelManagerEvents` should have a `shelf` event of type `shelfView`
- click on a product in the shelf
- in the `window.pixelManagerEvents` should have a `shelf` event of type `productClick`

#### Screenshots or example usage

#### Types of changes


Depends on: https://github.com/vtex-apps/product-summary/pull/286
